### DISCUSSION TITLE
Welcome to LaunchMenu Discussions!

### DISCUSSION THREAD

**Discussion by sancarn** [LaunchMenu/LaunchMenu#92] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by huanshiwushuang** [comment]
My windows+space button is occupied by language switching. LaunchMenu runs in the background, I don't know how to use it.
I have been looking for alternatives to autohotkey. I like the syntax of JavaScript. After I read the introduction on launchMenu's official website, I didn't understand the goal of launchMenu or what functions launchMenu had. So can you answer it for me? thanks.

I just want to use JavaScript to implement cross-platform global keyboard and mouse event monitoring.
[My English is very bad, the content comes from Google Translate]

**Reply by sancarn** [reply]
Hi, although LaunchMenu does expose an API to do that, LaunchMenu is a larger framework for building custom software (especially launched from LaunchMenu's search field).

By the sounds of it you are looking only to implement cross platform global keyboard shortcuts? In which case you should be using [node-global-key-listener](https://github.com/LaunchMenu/node-global-key-listener) ([npm](https://www.npmjs.com/package/node-global-key-listener)) directly for now 😊

**Reply by huanshiwushuang** [reply]
Thank you very much, your library is great, I have fulfilled my needs

**Reply by bramhoven** [reply]
@huanshiwushuang I was experiencing the same problem but using the shortcut `Windows + Shift + space` seemed to solve it for me. Hopefully this works for you as well 😄

**Reply by huanshiwushuang** [reply]
> @huanshiwushuang I was experiencing the same problem but using the shortcut `Windows + Shift + space` seemed to solve it for me. Hopefully this works for you as well 😄

Thank you, but my computer is still switching input methods.
I tried to delete other input methods, but only kept one, but it still doesn't work.
My system is Windows 10 Professional Edition 1909[18363.1556]

**Reply by bramhoven** [reply]
@huanshiwushuang Ah so the when using the shortcut windows switches languages instead of launching LM?

**Reply by huanshiwushuang** [reply]
> @huanshiwushuang Ah so the when using the shortcut windows switches languages instead of launching LM?

Google Translate is terrible, I can't understand what you mean.
However, I think LM needs to be able to customize shortcut keys

**Reply by sancarn** [reply]
> However, I think LM needs to be able to customize shortcut keys

You can. You can open LM by clicking on the icon in the tray. This will show LM window.

![image](https://user-images.githubusercontent.com/7938900/132399192-0ae703e6-f64b-4bb3-8895-27cf5136220a.png)

To change the setting search for: `s: open`

![image](https://user-images.githubusercontent.com/7938900/132398022-38456528-a98a-492a-96c7-129db625a180.png)

Press enter on `open launchmenu` and change the shortcut accordingly  😊
